site/layouts/splashpage.hbs

```diff
@@ -20,7 +20,7 @@
 		{{>headresources}}
 	</head>
 
-	<body class="{{pageclass}}" vocab="http://schema.org/" typeof="WebPage">
+	<body class="{{pageclass}}" vocab="http://schema.org/" resource="#wb-webpage" typeof="WebPage">
```

Comment:
Can you add the same RDFa attribute as the other page on the main element here at line 38